miio/tests/test_roidmivacuum_miot.py

```diff
@@ -0,0 +1,188 @@
+    # "uid": "12345678",
+    "volume": 4,
+    "water_level": WaterLevel.Mop,
+    "total_clean_time_sec": 321456,
+    "total_clean_areas": 345678,
+    "clean_counts": 987,
+}
+
+
+class DummyRoidmiVacuumMiot(DummyMiotDevice, RoidmiVacuumMiot):
+    def __init__(self, *args, **kwargs):
+        self.state = _INITIAL_STATE
+        super().__init__(*args, **kwargs)
+
+
+@pytest.fixture(scope="function")
+def dummyroidmivacuum(request):
+    request.cls.device = DummyRoidmiVacuumMiot()
+
+
+def assertEnum(a, b):
+    assert a == b
+    assert repr(a) == repr(b)
+
+
+@pytest.mark.usefixtures("dummyroidmivacuum")
+class TestRoidmiVacuum(TestCase):
+    def test_VacuumStatus(self):
+        status = self.device.status()
+        assert status.auto_boost == _INITIAL_STATE["auto_boost"]
+        assert status.battery == _INITIAL_STATE["battery_level"]
+        assertEnum(
+            status.charging_state, ChargingState(_INITIAL_STATE["charging_state"])
+        )
+        assertEnum(status.fanspeed, FanSpeed(_INITIAL_STATE["fanspeed_mode"]))
+        assert status.current_audio == _INITIAL_STATE["current_audio"]
+        assert status.clean_area == _INITIAL_STATE["clean_area"]
+        assert status.error_code == _INITIAL_STATE["error_code"]
+        assertEnum(status.state, RoidmiState(_INITIAL_STATE["state"]))
+        assert status.double_clean == _INITIAL_STATE["double_clean"]
+        assert status.edge_sweep == _INITIAL_STATE["edge_sweep"]
+        assert str(status.dnd_status) == str(
+            status._parse_forbid_mode(_INITIAL_STATE["forbid_mode"])
+        )
+        assert status.led_switch == _INITIAL_STATE["led_switch"]
+        assert status.lidar_collision == _INITIAL_STATE["lidar_collision"]
+        assert status.mop_present == _INITIAL_STATE["mop_present"]
+        assert status.is_mute == _INITIAL_STATE["mute"]
+        assert status.station_key == _INITIAL_STATE["station_key"]
+        assert status.station_led == _INITIAL_STATE["station_led"]
+        assertEnum(status.sweep_mode, SweepMode(_INITIAL_STATE["sweep_mode"]))
+        assertEnum(status.sweep_type, SweepType(_INITIAL_STATE["sweep_type"]))
+        assert status.timing == _INITIAL_STATE["timing"]
+        assertEnum(status.path_mode, PathMode(_INITIAL_STATE["path_mode"]))
+        assert status.progress == _INITIAL_STATE["progress"]
+        assert status.work_station_freq == _INITIAL_STATE["work_station_freq"]
+        assert status.volume == _INITIAL_STATE["volume"]
+        assertEnum(status.water_level, WaterLevel(_INITIAL_STATE["water_level"]))
+
+    def test_RoidmiCleaningSummary(self):
+        status = self.device.cleaning_summary()
+        assert (
+            status.total_duration.total_seconds()
+            == _INITIAL_STATE["total_clean_time_sec"]
+        )
+        assert status.total_area == _INITIAL_STATE["total_clean_areas"]
+        assert status.count == _INITIAL_STATE["clean_counts"]
+
+    def test_RoidmiConsumableStatus(self):
```

Comment:
```suggestion
    def test_consumable_status(self):
```
Avoid `CamelCasing`, it is also unnecessary to repeat "Roidmi" in tests.